File: client/src/app/_dtos/admin/forwardedOfficialDto.ts
export interface IForwardedOfficialDto
{
    checked: boolean;
    forwardCategoryId: number;
    officialName: string;
    agentName: string;
    dateTimeForwarded: Date;
    dateOnlyForwarded: Date;
    emailIdForwardedTo: string;
    phoneNoForwardedTo: string;
    whatsAppNoForwardedTo: string;
    loggedInEmployeeId: number;
}

export class forwardedOfficialDto implements IForwardedOfficialDto
{
    checked=false;
    forwardCategoryId: number=0;
    officialName: string='';
    agentName: string='';
    dateTimeForwarded: Date = new Date('1900-01-01');
    dateOnlyForwarded: Date = new Date('1900-01-01');
    emailIdForwardedTo: string='';
    phoneNoForwardedTo: string='';
    whatsAppNoForwardedTo: string='';
    loggedInEmployeeId: number=0;

}

File: api/DTOs/Admin/Orders/ForwardedOfficialDto.cs
namespace api.DTOs.Admin.Orders
{
    public class ForwardedOfficialDto
    {
        public int ForwardCategoryId { get; set; }
        public string OfficialName {get; set;}
        public string AgentName {get; set;}
        public DateOnly DateOnlyForwarded {get; set;}
        public string EmailIdForwardedTo { get; set; }
        public string PhoneNoForwardedTo { get; set; }
        public string WhatsAppNoForwardedTo { get; set; }
        public string Username {get; set;}
    }
}